boards/pulga/include/periph_conf.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2022 Universidade de São Paulo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_pulga
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the Pulga board
+ *
+ * @author      Geovane Fedrecheski <geonnave@gmail.com>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "kernel_defines.h"
+#include "periph_conf_common.h"
```

Comment:
```suggestion
#include "periph_cpu.h"
#include "cfg_clock_32_1.h"
#include "cfg_rtt_default.h"
#include "cfg_timer_default.h"
```

I think you want this - not the whole nRF52DK config.